docs/index.md

```diff
@@ -0,0 +1,41 @@
+Application Settings
+====================
+
+This scaffold brings a modified settings object to your project called `ApplicationSettings`. You can either
+use the class directly, however like all setting objects it's best if you extend the class and provide your
+own @property definitions on the doc comment to give your IDE some idea of what settings you're looking to
+support.
+
+The key difference with this settings object is that settings are persisted in a repository through the
+`ApplicationSetting` model (not singular not plural).
```

Comment:
Lol - typo. Should be "note singular not plural". It was just that there are 2 main classes in this module ApplicationSetting and ApplicationSettings - it was really just to make sure the unwary didn't just accept PHP Storm's first autocomplete suggestion....!